File: sandbox/python/src/TeamInvitesDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        response = api.TeamApi(api_client).team_invites(
            email_address=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Team#team_invites: %s\n" % e)

File: sandbox/php/src/TeamInvitesDefaultExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();

try {
    $response = (new Dropbox\Sign\Api\TeamApi(config: $config))->teamInvites(
        email_address: null,
    );

    print_r($response);
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling Team#teamInvites: {$e->getMessage()}";
}
